test/unit/test_local_mode.py

```diff
@@ -188,7 +188,7 @@ def test_v2_estimator_with_default_shots_option(self, backend):
         inst = EstimatorV2(backend=backend, options=options)
         job = inst.run(**get_primitive_inputs(inst, backend=backend))
         pub_result = job.result()[0]
-        self.assertDictEqual(pub_result.metadata, {"target_precision": 0.1})
+        self.assertIn(("target_precision", 0.1), pub_result.metadata.items())
```

Comment:
We could also check the `shots` included in the metadata (at least for this test)?

Reply:
Since the shots would only be included in Qiskit > 1.2, this test would fail on older versions